clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1825,6 +1825,26 @@ be applied multiple times to the same loop.
   }];
 }
 
+def SYCLDeviceIndirectlyCallableDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::device_indirectly_callable";
+  let Content = [{
+This attribute can only be applied to a functions and indicates that the
+function must be treated as device one and must be emitted even if it has no
```

Comment:
Perhaps "must be treated as a device function" is clearer

Reply:
Fixed, thanks